src/utils/account.ts

```diff
@@ -0,0 +1,495 @@
+// Initially copied from https://github.com/celo-org/celo-monorepo/blob/sdks-3.2.0/packages/sdk/cryptographic-utils/src/account.ts
```

Comment:
Is any part of this file meaningfully modified from the original? What of this ought to be reviewed, if any?

Reply:
Nothing was changed except some imports.